modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -619,8 +653,11 @@ struct InferROI: public cv::detail::KernelTag {
         // 0th is ROI, 1st is input image
         auto &&ii = uu.inputs.at(uu.params.input_names.at(0));
         auto &&mm = in_metas.at(1u);
-        configureInputInfo(ii, mm);
+        configureInputInfo(ii, const_cast<IEUnit*>(&uu), mm);
```

Comment:
Generally, `const_cast` is not good. It breaks the above contracts where we declare our inputs as `const`.

Reply:
Added FIXME.